### DISCUSSION TITLE
Is possible to compile Pixelorama on ubuntu?

### DISCUSSION THREAD

**Discussion by dacorsa** [Orama-Interactive/Pixelorama#926] [Q&A] [answered]
Hi all,
is possible to compile pixelorama on ubuntu mantic?

Can you write a little guide ? 
thanks best regards

**Comment by OverloadedOrama** [comment, marked as answer]
Hello! Yeah, it's possible. All you need to do is download the [latest stable version of Godot (currently 4.1.2)](https://godotengine.org/download/), download the Pixelorama source code from GitHub (either cloning it or downloading it as a zip should work fine, I recommend cloning it so you can stay up to date with the latest changes easily), open Pixelorama as a Godot project and run it. If you want to export it, you can read this https://docs.godotengine.org/en/stable/tutorials/export/exporting_projects.html.
